Samples.Objc/Samples.ObjC/API/PayPalAPI.m

```diff
@@ -40,7 +38,6 @@ - (id)init {
     self.baseURLv2 = @"https://api.sandbox.paypal.com/v2/";
     self.accessTokenEndpoint = [[FetchAccessTokenEndpoint alloc] init];
     self.createOrderEndpoint = [[CreateOrderEndpoint alloc] init];
```

Comment:
Should we store the `returnUrl` in the PayPalAPI singleton? so that when the user attempts to update their credentials, they are in one place? If that would involve larger changes to the local server that writes these values based on input from the user, then we can skip

Reply:
yep this is nice!!!! I'll do this :smile: